ignite/handlers/running_average.py

```diff
@@ -0,0 +1,66 @@
+from ignite.metrics import Metric
+from ignite.engine import Events
+
+
+class RunningAverage(object):
+    """Compute running average of a metric or output
+
+    Args:
+        src (str or Metric): input source: "output" string or an instance of :class:`ignite.metrics.Metric`
+        alpha (float, optional): running average decay factor, default 0.98
+        output_transform (Callable, optional): a function to use to transform the output if `src` is "output".
+            Otherwise it should be None
+
+    """
+
+    def __init__(self, src, alpha=0.98, output_transform=None):
+        if not (isinstance(src, Metric) or src == 'output'):
```

Comment:
Should we make the default `output` (or `None` meaning output)? I have no strong feeling about this, just an idea

Reply:
Good idea

Reply:
I like this idea